pkg/controlplane/storageversionhashdata/data.go

```diff
@@ -66,7 +66,8 @@ var GVRToStorageVersionHash = map[string]string{
 	"certificates.k8s.io/v1beta1/certificatesigningrequests":               "95fRKMXA+00=",
 	"coordination.k8s.io/v1beta1/leases":                                   "gqkMMb/YqFM=",
 	"coordination.k8s.io/v1/leases":                                        "gqkMMb/YqFM=",
-	"discovery.k8s.io/v1beta1/endpointslices":                              "Nx3SIv6I0mE=",
+	"discovery.k8s.io/v1/endpointslices":                                   "qgS0xkrxYAI=",
+	"discovery.k8s.io/v1beta1/endpointslices":                              "qgS0xkrxYAI=",
```

Comment:
storage version hash shouldn't change until 1.22

Reply:
changed storage version hash back to v1beta1